geoh5py/objects/drillhole.py

```diff
@@ -67,6 +71,11 @@ def __init__(self, object_type: ObjectType, **kwargs):
 
         super().__init__(object_type, **kwargs)
 
+        if vertices is None:
+            self._vertices = None
+        else:
+            self._vertices: np.ndarray = self.validate_vertices(vertices)
+
     @classmethod

```

Comment:
this is redundant with points that can be removed.
I would even suggest putting the default_type_uid mechanism in Object_Base, and return a trivial uid in this class, instead of repeating this function

Reply:
Done